### PR TITLE
Fixing MATI Ingestion Issues

### PR DESCRIPTION
**Pull request by chrishultin**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Fixes an issue where an incorrect sorting parameter was passed to the MATI API, resulting in IOCs failing to be ingested as expected.
Fixes an issue where an `mscore` of 0 would be treated as if the score was not present.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by chrishultin** [comment]
I'm still running some local ingestion testing to verify that everything completes successfully, but this _generally_ seems to be working.

**Comment by chrishultin** [comment]
> I'm still running some local ingestion testing to verify that everything completes successfully, but this _generally_ seems to be working.

Update: Tests appear to indicate that everything is functioning as expected.

File: Packs/MandiantAdvantageThreatIntelligence/Integrations/MandiantAdvantageThreatIntelligence/MandiantAdvantageThreatIntelligence.py [hunk 1]
```diff
@@ -1016,7 +1016,7 @@ def get_new_indicators(
     Returns:
         tuple[List, str]: A list of new indicators, and the new "last updated" checkpoint
     """
-    start_date = arg_to_datetime(last_run)
+    start_date = arg_to_datetime(last_run, None, True)
```
Comment (ilappe) [inline Packs/MandiantAdvantageThreatIntelligence/Integrations/MandiantAdvantageThreatIntelligence/MandiantAdvantageThreatIntelligence.py:1019]: the default of `is_utc` is `True`
so not need to change it

File: Packs/MandiantAdvantageThreatIntelligence/Integrations/MandiantAdvantageThreatIntelligence/MandiantAdvantageThreatIntelligence.py [hunk 2]
```diff
@@ -1028,16 +1028,20 @@ def get_new_indicators(
 
         param_start_date: datetime = datetime.fromtimestamp(0)
         if start_date is not None:
+            if start_date.tzinfo is None:
+                start_date = pytz.utc.localize(start_date)
+            else:
+                start_date = start_date.astimezone(pytz.UTC)
             param_start_date = max(
-                earliest_fetch.replace(tzinfo=pytz.UTC), start_date.replace(tzinfo=pytz.UTC)
+                earliest_fetch.replace(tzinfo=pytz.UTC), start_date
```
Comment (ilappe) [inline Packs/MandiantAdvantageThreatIntelligence/Integrations/MandiantAdvantageThreatIntelligence/MandiantAdvantageThreatIntelligence.py:1036]: is there is a logic different between this code and `max(param_start_date = max(
                earliest_fetch.replace(tzinfo=pytz.UTC), start_date.replace(tzinfo=pytz.UTC)`
?

**Review by ilappe — COMMENTED** [review]
Thanks @chrishultin for the contribution!
please see my small comments
thanks again!

File: Packs/MandiantAdvantageThreatIntelligence/Integrations/MandiantAdvantageThreatIntelligence/MandiantAdvantageThreatIntelligence.py [hunk 3]
```diff
@@ -1321,24 +1325,48 @@ def fetch_reputation(client: MandiantClient, args: dict = None):
...
+                markdown = tableToMarkdown(f'Mandiant Advantage Threat Intelligence'
+                                           f'information for {indicator["value"]}\n'
+                                           f'[View on Mandiant Advantage](https://advantage.mandiant.com/indicator/'
+                                           f'{indicator_type}/{indicator["value"]})',
+                                           table)
+
+                output.append(CommandResults(
+                    readable_output=markdown,
+                    outputs=indicator,
+                    outputs_prefix=f"MANDIANTTI.{input_type.upper()}",
```
Comment (ilappe) [inline Packs/MandiantAdvantageThreatIntelligence/Integrations/MandiantAdvantageThreatIntelligence/MandiantAdvantageThreatIntelligence.py:1348]: you should create **one** `CommandResults` object as results 
and all the indicators should be in the outputs
you can check `Packs/FeedDHS/Integrations/DHS_Feed/DHS_Feed.py:get_indicators_results` as reference

in addition, as this are out of the initial context of this PR
it's very recommended to open a new PR for that

Comment (chrishultin) [reply Packs/MandiantAdvantageThreatIntelligence/Integrations/MandiantAdvantageThreatIntelligence/MandiantAdvantageThreatIntelligence.py]: Got it - I'll move that fix over to a new PR shortly.

**Review by ilappe — APPROVED** [review]
Thanks @chrishultin!!

**Comment by ilappe** [comment]
instance test build fail due to the forked repo called `demisto-content` instead `content`

**Pull request merged**